.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
         - 'release/**'
+        - 'jbe/package_rename'
```

Comment:
This is a temporary change to test release building.  Will remove before merge.

Reply:
Removed in [ec470f4](https://github.com/awslabs/aws-sdk-swift/pull/678/commits/ec470f4180d897687bd00c9ab20ddea30e343362)